exporters/zipkin/zipkin.go

```diff
@@ -132,6 +143,15 @@ func (e *Exporter) ExportSpans(ctx context.Context, spans []sdktrace.ReadOnlySpa
 		return e.errf("failed to create request to %s: %v", e.url, err)
 	}
 	req.Header.Set("Content-Type", "application/json")
+
+	for k, v := range e.headers {
+		if k == "host" {
```

Comment:
This will not match the valid `"Host"` header.

```suggestion
		if strings.ToLower(k) == "host" {
```